lib/plugin.js

```diff
@@ -42,24 +42,26 @@ ManifestPlugin.prototype.apply = function(compiler) {
     var stats = compilation.getStats().toJson();
     var manifest = {};
 
-    _.merge(manifest, compilation.chunks.reduce(function(memo, chunk) {
-      var chunkName = chunk.name ? chunk.name.replace(this.opts.stripSrc, '') : null;
-
-      // Map original chunk name to output files.
-      // For nameless chunks, just map the files directly.
-      return chunk.files.reduce(function(memo, file) {
-        // Don't add hot updates to manifest
-        if (file.indexOf('hot-update') >= 0) {
+    _.merge(manifest, compilation.chunks
+      .filter(chunk => chunk.isInitial ? chunk.isInitial() : chunk.initial)
+      .reduce(function(memo, chunk) {
+        var chunkName = chunk.name ? chunk.name.replace(this.opts.stripSrc, '') : null;
+
+        // Map original chunk name to output files.
+        // For nameless chunks, just map the files directly.
+        return chunk.files.reduce(function(memo, file) {
```

Comment:
same as https://github.com/danethurber/webpack-manifest-plugin/pull/48/files#r126661193, but will work only in combination with the previous (e.g. if both functions are arrow)